Specs/Scene/GltfFeatureMetadataLoaderSpec.js

```diff
@@ -65,7 +65,7 @@ describe(
         ortho: {
           properties: {
             vegetation: {
-              type: "UINT8",
+              componentType: "UINT8",
               normalized: true,
             },
           },
```

Comment:
Just noticed something in this file (not in the diff): 

one of the tests should be renamed to be "throws if neither extension nor [extensionLegacy] is defined"